package.json

```diff
@@ -25,7 +25,9 @@
   "scripts": {
     "build": "ember build",
     "start": "ember server",
-    "test": "ember try:each"
+    "test": "ember test",
+    "test:all": "ember try:each",
+    "lint": "eslint addon-test-support config tests"
```

Comment:
this seems to be missing `*.js`

Reply:
Great point, I'll copy from my other PR in ember-cli.